src/components/SearchBox.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+class SearchBox extends React.PureComponent {
+	state = {
+		text: "",
+	};
+/*
+	handleSubmit = (e) => {
+		e.preventDefault();
+		console.log("Searching for:", this.state.text);
+	};
+*/
+   handleSubmit(e) {
+		e.preventDefault();
+		console.log("Searching for:", this.state.text);      
+   }
+
+	handleChange = (e) => {
+		this.setState({ text: e.target.value });
+	};
+
+	render() {
+		return (
+			<div>
+				<form onSubmit={(e) => this.handleSubmit(e)}>
```

Comment:
Optional: while `handleSubmit` as defined with functional declaration, you still "cheated" passing it as an arrow function. Try this version and see how you can make it work. See https://reactjs.org/docs/faq-functions.html#how-do-i-bind-a-function-to-a-component-instance
If you don't really understand why you can pass `handleSubmit` both ways and what the difference is, let me know and we'll have a chat.

```suggestion
				<form onSubmit={this.handleSubmit}>
```